.vscode/settings.json

```diff
@@ -1,3 +1,7 @@
 {
-    "dotnet-test-explorer.testProjectPath": "**/*Tests*.csproj"
-}
\ No newline at end of file
+    "dotnet-test-explorer.testProjectPath": "**/*Tests*.csproj",
+    "[csharp]": {
+        "editor.defaultFormatter": "csharpier.csharpier-vscode",
+        "editor.formatOnSave": true
```

Comment:
Sure everyone want wants format on save? I have it enabled myself in Rider, but some people might prefer doing it manually. 🤷‍♂️ As long as we run a check in Github for the correct formatting.

Reply:
True! The MSBuild package will ensure everything is formatted on build anyway. It's also possible to run formatting on git commit hook if we don't want formatting on build either, there's an example of that here: https://github.com/martinothamar/Mediator/blob/1ebfbd6ff73b5b19ff8ccaec2d66d81b3b1d9cc1/.husky/task-runner.json#L6

Reply:
I just don't like to discover formatting issues only when I've pushed the code, might as well make it impossible to not have it properly formatted 😄

Reply:
Yea, I've used hook before as well. And I always enable onSave.

Reply:
_“In matters of principle, stand like a rock; in matters of taste, swim with the current.” - Thomas Jefferson_

I'm sure there are things in CSharpier I don't like, but I prefer that we all use the same - and I can live with the things I don't like.